File: src/bot.js
const TelegramBot = require('node-telegram-bot-api');
const { start, help, answer } = require('./variables');
const { admins, token, chat } = require('./config');

let bot = new TelegramBot(token, { polling: true });

bot.onText(/\/start/, (msg) => {
  bot.sendMessage(msg.chat.id, start, {
    parse_mode: 'markdown'
  });
});

bot.onText(/\/help/, (msg) => {
  bot.sendMessage(msg.chat.id, help, {
    parse_mode: 'markdown'
  });
});

bot.on('message', (msg) => {
  bot.forwardMessage(chat, msg.chat.id, msg.message_id);
  bot.sendMessage(msg.chat.id, answer, {
    parse_mode: 'markdown'
  })
});

File: light_version/bot.py
import telebot
import logging
from variables import *


bot = telebot.TeleBot(TOKEN)


@bot.message_handler(commands=['start'])
def start_handler(message):
    bot.send_message(message.from_user.id, start_mess)


@bot.message_handler(commands=['help'])
def help_handler(message):
    bot.send_message(message.from_user.id, help_mess)


@bot.message_handler(func=lambda message: True)
def forward_handler(message):
    try:
        if message.chat.id == int(CHAT):
            bot.send_message(message.reply_to_message.forward_from.id, message.text)
        else:
            bot.forward_message(CHAT, message.chat.id, message.message_id)
    except Exception as error:
        print("Exception in forward handler. Info: {}".format(error))


def main(use_logging, level_name):
    if use_logging:
        telebot.logger.setLevel(logging.getLevelName(level_name))
    bot.polling(none_stop=True, interval=.5)


if __name__ == '__main__':
    main(True, 'DEBUG')
